Buildings/Controls/OBC/ASHRAE/FanCoilUnit/Subsequences/FanSpeed.mo

```diff
@@ -0,0 +1,423 @@
+within Buildings.Controls.OBC.ASHRAE.FanCoilUnit.Subsequences;
+block FanSpeed
+  "Fan speed setpoint subsequence"
+
+  parameter Boolean have_cooCoi
+    "Does the fan coil unit have a cooling coil? True: Yes, False: No";
```

Comment:
update the comment and set the default value to `true`.

Also, please propagate the condition to instances and parameters.